moveit_core/collision_detection/include/moveit/collision_detection/test_collision_common_panda.h

```diff
@@ -233,7 +233,7 @@ TYPED_TEST_P(CollisionDetectorPandaTest, DistanceSelf)
   collision_detection::CollisionResult res;
   this->cenv_->checkSelfCollision(req, res, *this->robot_state_, *this->acm_);
   ASSERT_FALSE(res.collision);
-  EXPECT_NEAR(res.distance, 0.13, 0.01);
+  EXPECT_NEAR(res.distance, 0.022, 0.001);
```

Comment:
I had to adapt this value, because the old ACM additionally disabled collisions between link5 and hand as well as link7:
| old | new |
|------|-------|
| ![image](https://user-images.githubusercontent.com/5376030/86897043-32d56880-c107-11ea-94fb-54070e9aef5e.png) | ![image](https://user-images.githubusercontent.com/5376030/86896665-98752500-c106-11ea-8251-f2175914016e.png) |